sample/F8/sessions

```diff
@@ -0,0 +1,1226 @@
+      "language":null,
+      "slides":null,
+      "audio":null,
+      "video":null,
+      "signup_url":null,
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
+         "id":1,
+         "name":"1 hour session",
+         "length":"01.00"
+      },
+      "track": null,
+      "start_time":"2017-04-18T18:00:00",
+      "end_time":"2017-04-18T19:00:00"
+   },
+   {
+      "short_abstract":null,
+      "long_abstract":null,
+      "id":2,
+      "title":"Breakfast",
+      "subtitle":"day-1-breakfast",
+      "microlocation":{
+         "id":2,
+         "name":"LL"
+      },
+      "comments": null,
+      "language":null,
+      "slides":null,
+      "audio":null,
+      "video":null,
+      "signup_url":null,
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
+         "id":2,
+         "name":"2 hour session",
+         "length":"02.00"
+      },
+      "track": null,
+      "start_time":"2017-04-18T08:00:00",
+      "end_time":"2017-04-18T10:00:00"
+   },
+   {
+      "short_abstract":null,
+      "long_abstract":null,
+      "id":3,
+      "title":"Keynote",
+      "subtitle":"day-1-keynote",
+      "microlocation":{
+         "id":3,
+         "name":"City National Civic"
+      },
+      "comments": null,
+      "language":null,
+      "slides":null,
+      "audio":null,
+      "video":null,
+      "signup_url":null,
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
+         "id":3,
+         "name":"3 hour session",
+         "length":"03.00"
+      },
+      "track": null,
+      "start_time":"2017-04-18T10:00:00",
+      "end_time":"2017-04-18T11:30:00"
+   },
+   {
+      "short_abstract":null,
+      "long_abstract":null,
+      "id":4,
+      "title":"Lunch",
+      "subtitle":"day-1-lunch",
+      "microlocation":{
+         "id":4,
+         "name":"Festival Hall"
+      },
+      "comments": null,
+      "language":null,
+      "slides":null,
+      "audio":null,
+      "video":null,
+      "signup_url":null,
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
+         "id":1,
+         "name":"1 hour session",
+         "length":"01.00"
+      },
+      "track": null,
+      "start_time":"2017-04-18T11:30:00",
+      "end_time":"2017-04-18T12:30:00"
+   },
+   {
+      "short_abstract":null,
+      "long_abstract":null,
+      "id":5,
+      "subtitle":"introduction-to-the-camera-effects-platform",
+      "title":"Introduction to the Camera Effects Platform",
+      "sessionDescription":"Come hear more about Camera Effects Platform, a suite of creative tools that lets you create effects for the Facebook camera, from simple photo frames to interactive masks and effects that use Augmented Reality (AR). In this session you’ll learn what camera effects are, how to create and submit your own, and how you can connect with the audiences that matter to you by providing rich experiences that help them express themselves.",
```

Comment:
@mahikaw what is this new `sessionDescription` field here ? This should move to either short or long abstract.